Clarify default PL monitor behavior (#33683)
DataDog/documentation: content/en/synthetics/platform/private_locations/monitoring.md

Monitors in the **Monitors** tab either have a group that corresponds to your private location ID or are tagged with `location_id:<ID_OF_THE_PL>`.

> > **Note:** These default monitors are created **only once**, when the first private location is created. If you previously had a private location that was deleted, creating a new private location later will **not** recreate the default monitors.  

Reviewer: ```suggestion
> **Note:** Default monitors are created **only once**, when the first private location is created. Deleting a private location and creating a new one does **not** recreate the default monitors.
```
